generators/server/files.js

```diff
@@ -556,12 +556,7 @@ const serverFiles = {
       ],
     },
     {
-      condition: generator =>
-        !generator.reactive &&
-        generator.authenticationType === 'oauth2' &&
-        (generator.applicationType === 'monolith' ||
-          generator.applicationType === 'microservice' ||
-          generator.applicationType === 'gateway'),
+      condition: generator => !generator.reactive && generator.authenticationType === 'oauth2' && generator.applicationType === 'monolith',
```

Comment:
What about gateway?

Reply:
This feature was never implemented for WebFlux. Since our gateway is now WebFlux-only, it doesn't make sense to include this. Adding it to WebFlux is being tracked by https://github.com/jhipster/generator-jhipster/issues/13461.

Reply:
This explains that. Thank you!